reV/supply_curve/supply_curve.py

```diff
@@ -19,6 +19,314 @@
 logger = logging.getLogger(__name__)
 
 
+class CompetitiveWindFarms:
```

Comment:
Move this into its own file? This one is breaking 1200 lines.

Reply:
done